MinnPost/object-sync-for-salesforce
# 204 handle large changesets

## What does this PR do?

This PR is intended to fix #204. It should process large batches according to the user's settings.

## How do I test this PR?

- Run a lot of changes - for example, in Salesforce you can create a temporary field on all the objects and increment it by one.

We need to make sure that it does follow the user's settings, and also that it does not cause performance issues.
